tests/_support/Page/price/Create.php

```diff
@@ -61,4 +71,31 @@ public function saveForm(): void
         $I->click('Save');
         $I->closeNotification('Prices were successfully created');
     }
+
+    /**
+     * @param string $text
+     */
+    public function fillXEditable(string $text): void
```

Comment:
Этот метод заполняет не просто какой-то xEditable, а конкретный – заметку.
```suggestion
    public function fillNote(string $text): void
```